java/src/test/java/org/rocksdb/ReadOptionsTest.java

```diff
@@ -144,6 +144,22 @@ public void iterateUpperBoundNull() {
     }
   }
 
+  @Test
+  public void iterateLowerBound() {
+    try (final ReadOptions opt = new ReadOptions()) {
+      Slice lowerBound = buildRandomSlice();
+      opt.setIterateLowerBound(lowerBound);
+      assertThat(Arrays.equals(lowerBound.data(), opt.iterateLowerBound().data())).isTrue();
+    }
+  }
+
+  @Test
+  public void iterateLowerBoundNull() {
+    try (final ReadOptions opt = new ReadOptions()) {
+      assertThat(opt.iterateLowerBound()).isNull();
+    }
+  }
+
   @Test
   public void copyConstructor() {
```

Comment:
Can you update this copyConstructor test as well with iterateLowerBound ? Also add fail* tests (see `failSetIterateUpperBoundUninitialized` and `failIterateUpperBoundUninitialized`).

Reply:
Added missing test cases.